printer/modules/gerard.py

```diff
@@ -5,10 +5,11 @@
 import logging
 import shlex
 import subprocess
-
+from datetime import datetime
```

Comment:
prefer importing the whole thing, and then update usage below
```suggestion
import datetime
```

Reply:
done